gtfs-validator/main/src/main/java/org/mobilitydata/gtfsvalidator/notice/RouteUniqueNamesNotice.java

```diff
@@ -37,7 +37,7 @@ public RouteUniqueNamesNotice(
             .put("comparedRouteCsvRowNumber", comparedRouteCsvRowNumber)
             .put("routeLongName", routeLongName)
             .put("routeShortName", routeShortName)
-            .put("routeType", routeType)
+            .put("routeType", routeType.toString())
```

Comment:
Is this a notice we created, or was it created by mobilitydata?

Reply:
It is a notice I created before.